packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/storage/data_export.py

```diff
@@ -23,51 +17,12 @@
 
 async def start_data_export(
     rabbitmq_rpc_client: RabbitMQRPCClient, *, paths: DataExportTaskStartInput
-) -> TaskRpcGet:
+) -> AsyncJobGet:
     result = await rabbitmq_rpc_client.request(
         STORAGE_RPC_NAMESPACE,
         _RPC_METHOD_NAME_ADAPTER.validate_python("start_data_export"),
         paths=paths,
         timeout_s=_DEFAULT_TIMEOUT_S,
     )
-    assert isinstance(result, TaskRpcGet)
-    return result
-
-
-async def abort_data_export(
-    rabbitmq_rpc_client: RabbitMQRPCClient, *, task_id: TaskRpcId
-) -> DataExportTaskAbortOutput:
-    result = await rabbitmq_rpc_client.request(
-        STORAGE_RPC_NAMESPACE,
-        _RPC_METHOD_NAME_ADAPTER.validate_python("abort_data_export"),
-        task_id=task_id,
-        timeout_s=_DEFAULT_TIMEOUT_S,
-    )
-    assert isinstance(result, DataExportTaskAbortOutput)
-    return result
-
-
-async def get_data_export_status(
-    rabbitmq_rpc_client: RabbitMQRPCClient, *, task_id: TaskRpcId
-) -> TaskRpcStatus:
-    result = await rabbitmq_rpc_client.request(
-        STORAGE_RPC_NAMESPACE,
-        _RPC_METHOD_NAME_ADAPTER.validate_python("get_data_export_status"),
-        task_id=task_id,
-        timeout_s=_DEFAULT_TIMEOUT_S,
-    )
-    assert isinstance(result, TaskRpcStatus)
-    return result
-
-
-async def get_data_export_result(
-    rabbitmq_rpc_client: RabbitMQRPCClient, *, task_id: TaskRpcId
-) -> TaskRpcResult:
-    result = await rabbitmq_rpc_client.request(
-        STORAGE_RPC_NAMESPACE,
-        _RPC_METHOD_NAME_ADAPTER.validate_python("get_data_export_result"),
-        task_id=task_id,
-        timeout_s=_DEFAULT_TIMEOUT_S,
-    )
-    assert isinstance(result, TaskRpcResult)
+    assert isinstance(result, AsyncJobGet)
```

Comment:
I would do something like this and make it reusable. 
We achive two things:
- we can easily resue it for let's say: copy of folder a folder, copy of a project, etc...
- we can more easily extract all this logic to a common library in the future

```suggestion
from datetime import timedelta
AsyncJobName: TypeAlias = str
async def starsubmit_job(
    rabbitmq_rpc_client: RabbitMQRPCClient, *, name: AsyncJobName, timeout: timedelta, **kwargs
) -> AsyncJobGet:
    result = await rabbitmq_rpc_client.request(
        STORAGE_RPC_NAMESPACE,
        TypeAdapter(RPCMethodName).validate_python(name),
        paths=paths,
        timeout_s=timeout.total_seconds(),
    )
    assert isinstance(result, AsyncJobGet)
```

Reply:
So even thouhg on the server part the `start_data_export` still needs to be defined as it currently is, on the client side we use this method and call it like so. This would be in the http API on the webserver.

```python
await submit_job(rabbit_rpc_client, name="start_data_export", paths=data_export_task_start_input)
```

Reply:
I see your point, but TBH I would prefer not to do this, if you agree. This function can reach any rpc endpoint exposed within the platform. I find it more intuitive that a single client function can hit a single server endpoint. In my opinion we should use something like openapi specs in the future to check compatibility between clients and servers also for rpc. And that would not work with this setup.

Reply:
We do not require openapi specs no, but we should use it (https://spec.open-rpc.org/). To me the specs are like type hinting in python, they allow you to automate checks that apis and clients are compatible or that apis don't break backwards compatibility. So I think we should use it also for the rpc. But that doesn't work if things are too dynamic. But ok if you are adamant to get have a general method I will add it.

Reply:
Done